docs/api-reference/classes/NSVersionInfo/SetCheckedInDate.md

```diff
@@ -1,19 +1,28 @@
 ﻿---
-uid: crmscript_ref_NSVersionInfo_SetCheckedInDate
-title: SetCheckedInDate(DateTime checkedInDate)
+uid: crmscript_class_nsversioninfo_setcheckedindate
+title: SetCheckedInDate()
+description: CRMScript method in the NSVersionInfo class that sets the date that this version was checked in
 intellisense: NSVersionInfo.SetCheckedInDate
-keywords: NSVersionInfo, GetCheckedInDate
+keywords: NSVersionInfo, GetCheckedInDate SetCheckedInDate(DateTime)
```

Comment:
I think there should be a comma between keywords

Reply:
done